### PR TITLE
[MIG] stock_picking_group_by_partner_by_carrier_by_date: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Depends on:
- stock_picking_group_by_partner_by_carrier https://github.com/OCA/stock-logistics-workflow/pull/795

### CODE REVIEW

**Review by mmequignon — APPROVED** [review]

File: stock_picking_group_by_partner_by_carrier_by_date/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Stock Picking: group by partner and carrier and scheduled date",
     "Summary": """Delivery orders will be matched by date too.
     """,
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (sebalix) [inline stock_picking_group_by_partner_by_carrier_by_date/__manifest__.py]: Should be `14.0.1.0.0`

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-stock_picking_group_by_partner_by_carrier @ git+https://github.com/phuctranerp/stock-logistics-workflow.git@14.0-mig-stock_picking_group_by_partner_by_carrier#subdirectory=setup/stock_picking_group_by_partner_by_carrier
+odoo14-addon-stock_move_assign_picking_hook @ git+https://github.com/phuctranerp/stock-logistics-workflow.git@14.0-mig-stock_move_assign_picking_hook#subdirectory=setup/stock_move_assign_picking_hook
```
Comment (sebalix) [inline test-requirements.txt]: These dependencies are now merged

**Review by sebalix — COMMENTED** [review]
LG, just the version number to fix.

**Comment by hailangvn** [comment]
We will work on this soon.

**Comment by hailangvn** [comment]
Thanks @sebalix. This PR has been rebased and updated.

**Review by sebalix — APPROVED** [review]

**Pull request merged**